libs/shared/enums/src/lib/utils/get-service-type.ts

```diff
@@ -3,16 +3,23 @@ import {
   ContainerApplicationEntity,
   DatabaseEntity,
   GitApplicationEntity,
+  JobApplicationEntity,
 } from '@qovery/shared/interfaces'
 import { ServiceTypeEnum } from '../service-type.enum'
 
 export const getServiceType = (data: ApplicationEntity | DatabaseEntity) => {
   let currentType: ServiceTypeEnum
 
+  const isJob = (data as JobApplicationEntity).max_nb_restart
+
```

Comment:
Ok so basically this file already exists, I think you could add some utils in this one that are: isDatabase(data) and that basically would check that getServiceType(data) === ServiceType.DATABASE you see?